recipe/meta.yaml

```
@@ -7,21 +7,21 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/deepmerge-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/deepmerge-{{ version }}.tar.gz
   sha256: 5c3d86081fbebd04dd5de03626a0607b809a98fb6ccba5770b62466fe940ff20
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python >=3.5
+    - python {{ python_min }}
     - setuptools-scm
   run:
-    - python >=3.5
+    - python >={{ python_min }}
```

```suggestion
    - python >={{ python_min }}
    - typing-extensions
```

This should add `typing_extensions` as a dependency. My guess is that this problem is precisely what the new `python_min` thing is supposed to uncover, that is, earlier builds where probably both build and testes in our CI with a recent python version, but since they where already `noarch` could be installed on older python versions, where they could fail due to the lack of the `typing_extensions` module.

---

Hi @zklaus, thanks for the suggestion! (In fact, I was going to ping you about the issue but got diverted to other things.)

I would have added the `typing-extensions` dependency, but missed `python_min`. So good to get your input!